spring-boot-project/spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -1009,9 +1009,9 @@ name of the bean is used.
 The bean name in the example above is `acme-com.example.AcmeProperties`.
 ====
 
-Even if the preceding configuration creates a regular bean for `AcmeProperties`, we
+The preceding configuration creates a regular bean for `AcmeProperties`. We
 recommend that `@ConfigurationProperties` only deal with the environment and, in
-particular, does not inject other beans from the context. Having said that, the
+particular, does not inject other beans from the context. Keep in mind that the
 `@EnableConfigurationProperties` annotation is _also_ automatically applied to your
 project so that any _existing_ bean annotated with `@ConfigurationProperties` is
 configured from the `Environment`. You could shortcut `MyConfiguration` by making sure
```

Comment:
Regarding the 'shortcut' in MyConfiguration:
In the original issue, @behrangsa asked: What is getting shortcut if we annotate MyConfiguration with @Component.

But I am reading the documentation and **AcmeProperties** class is the one annotated with @Component and not **MyConfiguration** class. 

Is my understanding correct - if we put the @Component annotation on AcmeProperties, we don't have to use the @EnableConfigurationProperties(AcmeProperties.class) in MyConfiguration class. Let me know and I can rephrase this sentence so that it does not lead to confusion.

Reply:
@viktoriya-kutsarova Yup, you're right. You either need to use `@EnableConfigurationProperties(AcmeProperties.class)` or annotate `AcmeProperties` with `@Component`.